misk/testing/src/main/kotlin/misk/testing/MiskTestExtension.kt

```diff
@@ -40,7 +40,7 @@ internal class MiskTestExtension : BeforeEachCallback, AfterEachCallback {
     override fun beforeEach(context: ExtensionContext) {
       if (context.startService()) {
         serviceManager.startAsync()
-        serviceManager.awaitHealthy(5, TimeUnit.SECONDS)
+        serviceManager.awaitHealthy(10, TimeUnit.SECONDS)
```

Comment:
lol oh hibernate

Reply:
I think it might be classpath scanning. Will try to improve, but it was hovering between 3 and 6 seconds.